lib/CL/pocl_llvm_build.cc

```diff
@@ -353,14 +355,45 @@ int pocl_llvm_build_program(cl_program program,
 
   POCL_MSG_PRINT_LLVM("all build options: %s\n", ss.str().c_str());
 
-  std::istream_iterator<std::string> begin(ss);
-  std::istream_iterator<std::string> end;
-  std::istream_iterator<std::string> i = begin;
-  std::vector<const char*> itemcstrs;
+  char WSReplacementChar = 0;
```

Comment:
The WS here was meant to denote "White Space", not a prefix for all variables. So WSTempOptions could be just TempOptions.